test/rules/no-focused-tests/tests.ts.lint

```diff
@@ -0,0 +1,15 @@
+ddescribe('myClass',() => {
+~~~~~~~~~                   [Forbidden focused test ddescribe]
+  iit('should do myMethod', () => {
+  ~~~                               [Forbidden focused test iit]
+    // run my test
+  });
+});
+
+fdescribe('myClass',() => {
+~~~~~~~~~                   [Forbidden focused test fdescribe]
+  fit('should do myMethod', () => {
+  ~~~                               [Forbidden focused test fit]
+    // run my test
+  });
+});
```

Comment:
Please also test...

* Correct uses of `describe("...", () => {...})` and `it("...", () => {...})`
* Variables and classes named `describe` and `it`
* 
    ```
    const it = "This is silly but technically valid";
    const describe = () => "This too!";

    describe();
    ```